### PR TITLE
KOGITO-6347: Enhance vscode-extension test to verify SVG filepath settings

### PR DESCRIPTION
**Pull request by zdrapela**
**Jira:** https://issues.redhat.com/browse/KOGITO-6347

### CODE REVIEW

File: packages/serverless-workflow-vscode-extension/it-tests/serverless-workflow-editor-extension-svg-filepath.test.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    const svgName = `${FILE_NAME_NO_EXTENSION}.svg`;
+
+    const editorWebViews = await testHelper.openFileFromSidebar(WORKFLOW_NAME, RESOURCE_FOLDER);
+
+    await testHelper.saveFileInTextEditor();
+
+    // verify SVG was generated after file save
+    const SVG_FILE_PATH: string = path.resolve(TEST_PROJECT_FOLDER, RESOURCE_FOLDER, svgName);
+    // assert(fs.existsSync(SVG_FILE_PATH), `SVG file was not generated at path: ${SVG_FILE_PATH}.`);
```
Comment (paulovmr) [inline packages/serverless-workflow-vscode-extension/it-tests/serverless-workflow-editor-extension-svg-filepath.test.ts]: @zdrapela Is this commented line still needed?

Comment (zdrapela) [reply packages/serverless-workflow-vscode-extension/it-tests/serverless-workflow-editor-extension-svg-filepath.test.ts]: It is not, will remove that. Thanks

File: packages/serverless-workflow-vscode-extension/it-tests/serverless-workflow-editor-extension-svg-filepath.test.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    const svgName = `${FILE_NAME_NO_EXTENSION}.svg`;
+
+    const editorWebViews = await testHelper.openFileFromSidebar(WORKFLOW_NAME, RESOURCE_FOLDER);
+
+    await testHelper.saveFileInTextEditor();
+
+    // verify SVG was generated after file save
+    const SVG_FILE_PATH: string = path.resolve(TEST_PROJECT_FOLDER, RESOURCE_FOLDER, svgName);
+    expect(fs.readFileSync(SVG_FILE_PATH, "utf-8")).to.match(
+      new RegExp("<svg.*<\\/svg>"),
```
Comment (domhanak) [inline packages/serverless-workflow-vscode-extension/it-tests/serverless-workflow-editor-extension-svg-filepath.test.ts]: nitpick: add this as a constant to clearly describe what the REGEX does, for example - `IS_VALID_SVG_REGEX`

**Review by domhanak — APPROVED** [review]
@zdrapela one small nitpick regarding readability from me,  otherwise a great addition to tests! Thanks! I am approving, the nitpick can be considered or not.

File: packages/serverless-workflow-vscode-extension/it-tests/serverless-workflow-editor-extension-svg-filepath.test.ts (hunk 2 above)
Comment (zdrapela) [reply packages/serverless-workflow-vscode-extension/it-tests/serverless-workflow-editor-extension-svg-filepath.test.ts]: Your nitpick is welcome. Thanks!

I also noticed I left there some unused imports, so I am removing them.

**Review by paulovmr — APPROVED** [review]

**Pull request merged**